code/parachain/frame/staking-rewards/src/lib.rs

```diff
@@ -37,7 +37,7 @@
 #[cfg(any(feature = "runtime-benchmarks", test))]
 mod benchmarking;
 mod prelude;
-#[cfg(test)]
+#[cfg(any(feature = "runtime-benchmarks", test))]
```

Comment:
This seems to be breaking benchmarks. What was the reasoning for this being added?

Reply:
It actually fixed benchmarks locally for me, very strange. I'll review it again

Reply:
modification is correct afaik.

Reply:
These should be fixed now - just have to wait for benchmarks to run to be sure :+1: